targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/TARGET_UBLOX_C030/battery_charger_i2c.cpp

```diff
@@ -0,0 +1,33 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "battery_charger_i2c.h"
+
+BatteryChargerI2c::BatteryChargerI2c(PinName sda, PinName scl):_i2c(), _hz(100000) {
```

Comment:
`{` on the new line

Reply:
@0xc0170 C030 is based on STM32F4 so mbed_sdk_init is already defined as a strong function in mbed_overrides.c located in TARGET_STM. The HAL_MspInit function is called from within mbed_sdk_init and is meant to be used in user file as per stm32f4xx_hal.c The following comment is extracted from stm32f4xx_hal.c referring to HAL_MspInit:
  /* NOTE : This function Should not be modified, when the callback is needed,
            the HAL_MspInit could be implemented in the user file
   */

Change in the coding style noted.

Reply:
Can you do that change?

Reply:
Apologies...change commited